README.md

```diff
@@ -1,369 +1,168 @@
-  - provide 100% uptime to relay transactions
-  - listen for Deposit events on Home bridge to mint erc20 token on Foreign bridge
-  - listen for Withdraw events on Foreign bridge to unlock funds on Home Bridge
-- User role:
-  - sends POA coins to Home bridge in order to receive ERC20 token on Foreign Bridge using the same address to send/receive
-  - sends ERC20 POA20 token on Foreign Bridge in order to receive POA coins on Home Bridge using the same address to send/receive
-
-## Dependencies
-
-- [poa-bridge-contracts](https://github.com/poanetwork/poa-bridge-contracts/tree/0.1)
+### User Transactions
+- Connect to the network you want to transfer coins from using a web3 wallet such as Nifty Wallet or MetaMask. This can be the Home or Foreign network. The wallet must be funded to include amount to transfer and any gas costs.
+- Specify the amount to send.
+- Click the Transfer button.
+- Confirm the transaction via the web3 wallet. 
+- The same address is used to send a coin from the Home network and receive a token on the Foreign Network.
+
+### Resources
+Resources that describe various versions of the bridge. Some may be outdated, but will give a general sense of the UI and transactional flow.
+
+- [Deployed URL for POA -> Ethereum Network Bridge](https://bridge.poa.net/)
+- [Testnet Bridge URL](https://bridge-testnet.poa.net/)
+- [Bridge UI Tutorial Videos](https://www.youtube.com/playlist?list=PLS5SEs8ZftgUqR3hVFiEXQLqE9QI8sIGz)
+- [Article on the POA Bridge](https://medium.com/poa-network/cross-chain-bridges-paving-the-way-to-internet-of-blockchains-422ac94bc2e5)
+
+
+## Getting Started
+
+The following is an example setup using the POA Sokol testnet as the Home network, and the Ethereum Kovan testnet as the Foreign network. The instructions for the Bridge UI are identical for an `ERC20-to-ERC20` configuration, but the smart contract deployment steps will vary.
+
+### Dependencies
+
+- [poa-bridge-contracts](https://github.com/poanetwork/poa-bridge-contracts)
+- [token-bridge](https://github.com/poanetwork/token-bridge)
 - [node.js](https://nodejs.org/en/download/)
-- [metamask](https://metamask.io/)
-- happy mood and patience
+- [Nifty Wallet](https://chrome.google.com/webstore/detail/nifty-wallet/jbdaocneiiinmjbjlgalhcelgbejmnid?hl=en) or [MetaMask](https://metamask.io/)
+
+### Example Setup
 
-## Preparation for using the UI App
-1. Create empty folder where you will be setting up your bridge. In the following example lets call it `sokol-kovan-bridge`  
+1. Create an empty folder where you will be setting up your bridge. In this example we call it `sokol-kovan-bridge`.
 `mkdir sokol-kovan-bridge && cd sokol-kovan-bridge`  
 
-### 2. Deploy Sokol <-> Kovan Bridge contracts:  
-Prepare temporary eth address for deployment
-```
-DEPLOYMENT_ACCOUNT_ADDRESS=0x08660156928d768D26D3eeF642c11527FDca0891
-DEPLOYMENT_ACCOUNT_PRIVATE_KEY=67..14
-```
+2. Prepare temporary ETH address(es) for deployment. Each validator will require an address. 
```

Comment:
You don't need a JSON keystore for validators since the Token Bridge works with the private keys. So, the step to get the validator's private key could be to export the private key from the wallet. So, the step 3 is not needed in this case.